VimeoNetworking/VimeoNetworking/Request.swift

```diff
@@ -223,14 +167,37 @@ public struct Request<ModelType: MappableResponse>
     
     internal func associatedPageRequest(newPath newPath: String) -> Request<ModelType>
     {
+        // Since page response paging paths bake the paging parameters into the path (even if we originally provide them in the body),
```

Comment:
See the below comment. I'm going to modify this to keep the body intact and update the paging values with the updated values provided in the query params rather than keeping the url intact and deleting duplicate params in the body.